Sources/StreamChat/Database/DTOs/ChannelConfigDTO.swift

```diff
@@ -24,8 +24,7 @@ final class ChannelConfigDTO: NSManagedObject {
     @NSManaged var commands: NSOrderedSet
 
     func asModel() throws -> ChannelConfig {
-        guard isValid else { throw InvalidModel(self) }
```

Comment:
Can we also delete `isValid` from our extension of `NSManagedObject`?

Reply:
Trying to understand what's the impact of this change. I see `isValid` just checks if there's an existing object with that identifier. Now, even if the object was potentially deleted, the data in memory is still somehow available, and the corresponding model would just be populated, right?

Reply:
Yes. The thing is that, worse it can't be than this. If isValid is false and it reaches this point, it crashes. So if it does not crash at this point, at most it can crash later, either way having this is the worst scenario, so better to remove it

Reply:
@laevandus We cant remove it, since it is used in other places